source/usage-examples/findOne.txt

```diff
@@ -25,7 +25,8 @@ Click `here <{+example+}/findOne.go>`__ to see a fully runnable example.
 Expected Result
 ---------------
 
-After you run the code example, you should see output similar to:
+After you run the preceding example, it returns a ``SingleResult``
+object with the following document:
```

Comment:
I think "with" is ambiguous. If SingleResult contains documents, we should specify that. Otherwise, it could be interpreted as "alongside" meaning both a SingleResult object and the following documents are returned (in some unspecified object/format).